packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -23,13 +24,15 @@ export interface OriginalRequest {
  * @property type - The json-prc signing method for which a signature request has been made.
  * A 'Message' which always has a signing type
  * @property rawSig - Raw data of the signature request
+ * @property securityProviderResponse - Response from a security provider, whether it is malicious or not
  */
 export interface AbstractMessage {
   id: string;
   time: number;
   status: string;
   type: string;
   rawSig?: string;
+  securityProviderResponse?: Json;
```

Comment:
As we know this is always an object, would `Map<string, Json>` be more accurate?

Reply:
yes, I just pushed a fix.